cypress/integration/oauth2/consent.js

```diff
@@ -36,6 +36,34 @@ describe('OAuth 2.0 End-User Authorization', () => {
       .then(body => {
         expect(body.length).to.be.greaterThan(0);
         expect(hasConsent(client, body)).to.be.true;
+        body.forEach(consent => {
```

Comment:
`handled_at` should probably already been set - even without this code, right? `sessions/consent` should only show "handled" consent requests - which implies that they need the `handled_at` to be set.

Reply:
You're right. We have updated the test.